### PR TITLE
Build both target platforms in the pipeline

### PR DESCRIPTION
**Pull request by merks**

### CODE REVIEW

**Comment by merks** [comment]
@cdietrich 

Because this is a Jenkinsfile change, I don't think the verification build will test it.  Please work your magic...

**Comment by cdietrich** [comment]
pushed to https://ci.eclipse.org/xpect/job/Xpect/job/merks-pr-built-targets/

File: Jenkinsfile [hunk 1]
```diff
@@ -78,15 +72,37 @@ pipeline {
...
+                -Dtarget-platform=${TARGET_PLATFORM_LATEST} \
+                clean \
+                integration-test
+              '''
+          }
+        }
+      }
+    }
+
+    stage('Build With Primary') {
```
Comment (cdietrich) [inline Jenkinsfile:105]: do we need a clean -fdx or something the like between the steps?

Comment (cdietrich) [reply Jenkinsfile:105]: (in the old Jenkijnsfile we used a separate checkout location)

Comment (merks) [reply Jenkinsfile:105]: I don't think so.  Both do mvn clean.  Neither do an install so the maven cache can be reused.

**Review by cdietrich — APPROVED** [review]

**Comment by cdietrich** [comment]
thx @merks

**Comment by merks** [comment]
@cdietrich 

Could you please go here:

https://ci.eclipse.org/xpect/job/Xpect/job/master/

and kick off a build, choosing "milestone"

image

**Comment by cdietrich** [comment]
https://ci.eclipse.org/xpect/job/Xpect/job/master/770/

**Comment by cdietrich** [comment]
seems to have worked
https://download.eclipse.org/xpect/updates/

**Comment by merks** [comment]
Very good.  Thanks!

**Pull request merged**